src/modules/returns/lib/helpers.js

```diff
@@ -96,6 +94,30 @@ const getLicenceReturns = async (licenceNumbers, page = 1, isInternal = false) =
   return { data, pagination };
 };
 
+/**
+ * Checks whether user uses XML Upload for a list of licence numbers
+ * @param {Array} licenceNumbers to check if isUpload flag is true
+ * @param {Boolean} isInternal to specify user type
+ * @return {Boolean} if user has XML Upload functionality
```

Comment:
As it's an async function it returns a promise.  Having had a quick look at JSDoc docs, I think this should probably be:
```
@return {Promise<boolean>}
```
Which I haven't been doing...  I have only been doing @return {Promise}